src/server/api/file/getMultipleS3Files.ts

```diff
@@ -0,0 +1,44 @@
+import * as path from 'path'
```

Comment:
Added support for fetching multiple files, should we omit this change instead? we are not using it currently @minotogna 

(I thought bulk download would need it so I added but on closer look it generated the needed files on the fly 👍 )